docs/UserGuide.adoc

```diff
@@ -12,134 +12,340 @@ ifdef::env-github[]
-* Parameters can be in any order e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+* The first word in a command is the command word. The rest of the words are parameters, which
+are either compulsory or optional. +
+e.g. in a command `add n/**NAME** p/**PHONE**`, `add` is the command word while `n/**NAME** p/**PHONE**`
+are parameters.
+* Words in bold are the parameters to be supplied by the user +
+e.g. in `add **n/NAME**`, `**NAME**` is a parameter which can be used as `add n/John Doe`.
+* `[attributes]` refer to all the attributes in the <<List of Attributes>>.
+* Parameters can be in any order +
+e.g. if the command specifies `n/**NAME** p/**PHONE**`, `p/**PHONE** n/**NAME**`
+is also acceptable.
 ====
 
+=== List of attributes
+
+In this application, a person's data is represented by attributes. Attributes
+refer to personal particulars such as name, type of person and blood type. +
+
+
+The following list shows the list of attributes that can be specified for
+a person and the criteria for valid inputs:
+
+* n/**NAME**: the name of the person
+** Applicable to: patients, doctors, donors
+** Valid input: alphabets
+
+* t/**TYPE**: the type of person
+** Applicable to: patients, doctors, donors
+** Valid input: patient/donor/doctor
+
+* o/**ORGAN**: the type of organs
+** Applicable to: patients, donors
+** Valid input: kidney
+
+* b/**BLOODTYPE** - the blood type of the person
+** Applicable to: patients and donors
+** Valid input: A/B/AB/O
+
+* p/**PHONE** - contact number of a person
+** Applicable to: patients, doctors, donors
+** Valid input: 9 digit number sequence, starting with 6, 8 or 9
+
+* ic/**NRIC** - NRIC number of a person
+** Applicable to: patients, doctors and donors
+** Valid input: any sequence of alphabets and numbers that fulfill these criteria:
+*** starts with a capital S, F, T, or G
+*** followed by a 7 digit number sequence
+*** ends with a capital letter
+
+* pr/**PRIORITY** - priority of the patient, the value of which is determined by a doctor
+** Applicable to: patients
+** Valid input: high/medium/low
+
+* a/**AGE** - age of a person
+** Applicable to: patients and donors
+** Valid input: a number
+
+* d/**DOCTOR** - doctor in charge of a patient
+** Applicable to: patients
+** Valid input: NRIC of the doctor
+
+* tt/**TISSUE TYPE** - the type of tissues
+** Applicable to: patients and donors
+** Valid input: a string of 6 numbers (from 1 - 12), separated by commas
+
+* exp/**EXPIRY** - expiry date of a donor's organ
+** Applicable to: donors
+** Valid input: a valid date in DD-MMM-YYYY format
+
 === Viewing help : `help`
 
 Format: `help`
 
 === Adding a person: `add`
 
-Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a patient, doctor, or donor to OTM with the attributes given, depending on what is specified.
+
+There are two ways to add a person:
+* Normal Add
```

Comment:
what is this `*` supposed to do, in the netflify, its not changing the font style

Reply:
It is a bullet point